### ISSUE TITLE
[FEATURE] Goals with several metrics

### ISSUE THREAD

**Issue by gerardgn** [Leantime/leantime#1853]
In the OKR definition, The goals is measured by several Key result

I would like to add several metrics in the goals 

![Leantime_multiple metrics in goal for OKR](https://github.com/Leantime/leantime/assets/66369417/4fe27dc0-c7af-4e14-8678-9a04ed0255da)

**Comment by gloriafolaron** [comment]
Are you currently solving this by creating multiple goals with the different metric?

**Comment by gerardgn** [comment]
Yes I have to create multiple goals with different metric instead of one goal with different metrics. It is restrictive.
